docs/traditional-media.md

```diff
@@ -1106,54 +1106,72 @@ category 对应的关键词有
+    |      | aerospace-defense   | autos-transportation   | energy | environment | finance | healthcare-pharmaceuticals   | media-telecom   | retail-consumer   | sustainable-business | charged | future-of-health | futrue-of-money | take-five | reuters-impact |
 
-    -   资讯子频道：
+-   legal/ `:category`:
 
-    | 中国财经  | 国际财经                  | 新闻人物      | 财经视点     |
-    | ----- | --------------------- | --------- | -------- |
-    | china | internationalbusiness | newsmaker | opinions |
+    | All | Goverment | Legal Industry | Litigation | Transaction |
+    | --- | --------- | -------------- | ---------- | ----------- |
+    |     | goverment | legalindustry  | litigation | transaction |
 
-    -   专栏子频道：
 
-    | 中国财经专栏   | 国际财经专栏    | 大宗商品专栏    |
-    | -------- | --------- | --------- |
-    | CnColumn | IntColumn | ComColumn |
+</Route>
 
--   美国分站 `us`：
+### 订阅作者
 
-    -   主频道：
+<Route author="LyleLee HenryQW proletarius101" example="/reuters/authors/reuters" path="/reuters/authors/:category?" :paramsDesc="['category, 设置为作者名字, 默认作者是 reuters']">
 
-    | Business | Markets | World | Politics | Tech       | Breakingviews | Wealth | Life      |
-    | -------- | ------- | ----- | -------- | ---------- | ------------- | ------ | --------- |
-    | business | markets | world | politics | technology | breakingviews | wealth | lifestyle |
+-   authors/`:category`:
 
--   英国分站 `uk`：
+    | 默认    | Jonathan Landay | 其他作者          |
+    | ------- | --------------- | ----------------- |
+    | reuters | jonathan-landay | 其他作者名字       |
 
-    -   主频道：
+</Route>
 
-    | Business | Markets | World | UK | Tech       | Money           | Breakingviews | Sport  | Life      |
-    | -------- | ------- | ----- | -- | ---------- | --------------- | ------------- | ------ | --------- |
-    | business | markets | world | uk | technology | personalFinance | breakingviews | sports | lifestyle |
+### 深度调查栏目
+
+<Route author="LyleLee HenryQW proletarius101" example="/reuters/inverstigates" path="/reuters/inverstigates" :paramsDesc="[]">
 
 </Route>
 
+## RTHK
+
+### News
+
+RTHK offical provides full text RSS, check the offical website for detail information: <https://news.rthk.hk/rthk/en/rss.htm>
+
+This route adds the missing photo and Link element. (Offical RSS doesn't have Link element may cause issue on some RSS client)
+
+<RouteEn author="KeiLongW" example="/rthk-news/hk/international" path="/rthk-news/:lang/:category" :paramsDesc="['Language，Traditional Chinese`hk`，English`en`','Category']">
+
+| local      | greaterchina       | international | finance      | sport      |
+| ---------- | ------------------ | ------------- | ------------ | ---------- |
+| Local News | Greater China News | World News    | Finance News | Sport News |
+
+</RouteEn>
+
```

Comment:
```suggestion
```

Reply:
Oh, thanks to point out. Did not mean to change this paragraph.